### PR TITLE
Update qalculate-gtk.desktop.in and install: add translations

### PR DESCRIPTION
**Pull request by kuraga**

### CODE REVIEW

**Comment by hanna-kn** [comment]
Translations should not be added to `qalculate-gtk.desktop.in`. Translations are automatically added to `qalculate-gtk.desktop` (generated with qalculate-gtk.desktop.in as template) from the files in the `po` directory.

**Comment by kuraga** [comment]
The same for `data/pkg/install`?

**Comment by hanna-kn** [comment]
> The same for data/pkg/install?

No, the data in `install` is a manual copy of qalculate-gtk.desktop. Since the package (self-contained static binaries) this is used for does not support translations within the program, I did not deem it necessary to include translations in the desktop file.

**Comment by kuraga** [comment]
I haven't understand whether https://github.com/Qalculate/qalculate-gtk/pull/530/commits/7b7676afb7f5923d543f68bdb0638b83f0d3e775 is needed. Feel free to close, or pick up. Thanks!

**Comment by kuraga** [comment]
> > The same for data/pkg/install?
> 
> No, the data in `install` is a manual copy of qalculate-gtk.desktop. Since the package (self-contained static binaries) this is used for does not support translations within the program, I did not deem it necessary to include translations in the desktop file.

@hanna-kn , so is the (isolately) https://github.com/Qalculate/qalculate-gtk/commit/7b7676afb7f5923d543f68bdb0638b83f0d3e775 commit needed?

**Comment by hanna-kn** [comment]
I will leave it as it is for now.

**Comment by hugok79** [comment]
`pt_PT` translation is missing...

File: data/pkg/install [hunk 1]
```diff
@@ -35,6 +35,16 @@ Version=1.0
 Name=Qalculate!
 GenericName=Calculator
 Comment=Powerful and easy to use calculator
```
Comment (ovari) [inline data/pkg/install:37]: Please add [hu](https://github.com/Qalculate/qalculate-gtk/issues/554)

`Comment[hu]=Erőteljes és könnyen használható számológép`

Cf. https://github.com/Qalculate/qalculate-gtk/pull/555

File: data/pkg/install [hunk 2]
```diff
@@ -44,6 +54,14 @@ StartupNotify=true
 StartupWMClass=qalculate
 Categories=GTK;Utility;Calculator;Science;Math;
 Keywords=calculation;arithmetic;scientific;financial;
```
Comment (ovari) [inline data/pkg/install:56]: Please add [hu](https://github.com/Qalculate/qalculate-gtk/issues/554)

`Keywords[hu]=számítás;számológép;tudományos;pénzügyi;`

Cf. https://github.com/Qalculate/qalculate-gtk/pull/555

File: data/qalculate-gtk.desktop.in [hunk 3]
```diff
@@ -3,6 +3,16 @@ Version=1.0
 _Name=Qalculate!
 _GenericName=Calculator
 _Comment=Powerful and easy to use calculator
```
Comment (ovari) [inline data/qalculate-gtk.desktop.in:5]: Please add [hu](https://github.com/Qalculate/qalculate-gtk/issues/554)

`_Comment[hu]=Erőteljes és könnyen használható számológép`

Cf. https://github.com/Qalculate/qalculate-gtk/pull/555

File: data/qalculate-gtk.desktop.in [hunk 4]
```diff
@@ -12,3 +22,11 @@ StartupNotify=true
 StartupWMClass=qalculate-gtk
 Categories=GTK;Utility;Calculator;Science;Math;
 _Keywords=calculation;arithmetic;scientific;financial;
```
Comment (ovari) [inline data/qalculate-gtk.desktop.in:24]: Please add [hu](https://github.com/Qalculate/qalculate-gtk/issues/554)

`_Keywords[hu]=számítás;számológép;tudományos;pénzügyi;`

Cf. https://github.com/Qalculate/qalculate-gtk/pull/555

**Review by ovari — COMMENTED** [review]
Thank you

**Comment by kuraga** [comment]
@ovari , done. Thanks!

**Comment by kuraga** [comment]
@hanna-kn  ,why?

**Comment by hanna-kn** [comment]
> @hanna-kn ,why?

I closed to pull request primarily to discourage misguided updating of `qalculate.desktop.in`. As I've said before, this file is translated using the relevant text strings in the po files. `qalculate.desktop` (without .in) is the file that is installed. If I merged this patch, all translated text strings from `qalculate.desktop.in` would appear (as text strings for translation) in every po file, regardless of language.

The `install` file is only used for the static GNU/Linux binaries, which does not include any translations (I do no not if it somehow would be possible to compile in the translations). If I someday would decide to include translations for the desktop file in these packages, a simple copy and paste of the complete `qalculate.desktop` file is all that is required.

**Pull request closed without merging**